src/animations/Wiggle.js

```diff
@@ -0,0 +1,75 @@
+import React, { PureComponent } from 'react';
+import autoBindReact from 'auto-bind/react';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import { Animated, Easing } from 'react-native';
+
+export default class Wiggle extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    autoBindReact(this);
+
+    this.animation = new Animated.Value(0);
+  }
+
+  shouldComponentUpdate(nextProps) {
```

Comment:
nein !

Use ComponentDidUpdate, but compare previous with current props. We want to trigger animation if the prop was false and was set to true. This shouldn't fall into loop unless you're making other manipulations inside some of the parent, which is also a kind of an anti-pattern

Reply:
Brainfarted again 😂